src/runtime/vdom/vdom-render.ts

```diff
@@ -87,16 +87,21 @@ const createElm = (oldParentVNode: d.VNode, newParentVNode: d.VNode, childIndex:
     if (BUILD.svg && !isSvgMode) {
       isSvgMode = newVNode.$tag$ === 'svg';
     }
+
+    if (!win.document) {
+      throw new Error('You are trying to render a component in a non-browser environment. This is not supported.');
```

Comment:
total nit - but, IDK if this is a completely accurate statement (so slightly misleading) because we do render components in a non-browser environment via MockDoc?

Reply:
Good call, fixed.